.fixtures.yml

```diff
@@ -20,3 +20,4 @@ fixtures:
     facts: 'git://github.com/puppetlabs/puppetlabs-facts.git'
     puppet_agent: 'git://github.com/puppetlabs/puppetlabs-puppet_agent.git'
     provision: 'git://github.com/puppetlabs/provision.git'
+    servicenow_tasks: 'git://github.com/puppetlabs/servicenow_tasks.git'
```

Comment:
@gsparks We should probably tag this. I guess I could also just point to a specific commit.